lib/dash/mpd_utils.js

```diff
@@ -269,8 +269,12 @@ shaka.dash.MpdUtils.createTimeline = function(
 
     for (var j = 0; j <= repeat; ++j) {
       var endTime = startTime + d;
-      timeline.push(
-          {start: (startTime / timescale), end: (endTime / timescale)});
+      item = {
```

Comment:
I add `var item = {` in here, rebuild Shaka player with your code and it works fine. I test without your code, the issue appears again.  Thanks @duynguyen-ew . How can we propose your change to apply in Shaka player repository.